### ISSUE TITLE
Inter module dependency not working for Fragment

### ISSUE THREAD

**Issue by hengsin** [JaneWardSandy/eclipse-pde-partial-idea#72]
**Describe the bug**
The inter module dependency inspection doesn't check the Fragment-Host relation define at MANIFEST.MF

**Expected behavior**
Plugin added dependency between the fragment and fragment-host module.

**Comment by JaneWardSandy** [comment]
Emmmm, I had never developed the `Fragment plugins` before, could you please attach the demo?

Ex.
`Host A` export package a,
`Fragment module B` was the extension of `Host A`, and it export package a, b.

`Module C` import package a, resolve should add `Host A` and `Fragment module B` into dependency list.
Order list was:
xxx other dependency xxx
`Fragment module B`
`Host A`
xxx other dependency xxx

Is it like this?

**Comment by hengsin** [comment]
Hi,
I've not tested the scenario you describe above but yes, that's one of the expected behavior.

The scenario that I've tested is:
- Host A export package a,
- Fragment module B is the extension of Host A and it uses package a. Since module B is a Fragment, it doesn't need to add Import-Package a to access package a.
- Resolve should add Host A into dependency list of module B. That's not happening now.

**Comment by hengsin** [comment]
You get it the other way round, the issue here is not the export from the fragment but the export from the fragment-host. In other word, all packages exported from fragment host should be visible to the fragment module without the need for the fragment module to explicitly import those packages. For this to work, the plugin need to auto add the "Fragment Host" module into the dependency list of the "Fragment" module.

In Short:
- Module 1 is Fragment Host
- Module 2 is Fragment
- The dependency tab of Module 2 should have Module 1. This is not added by the plugin now.

**Comment by JaneWardSandy** [comment]
I got it.
Currently this plugins not support Fragment, I will check and add the commonly used function for Fragment.

**Comment by JaneWardSandy** [comment]
This issue has been fix and release at [v1.3.6.2](https://github.com/JaneWardSandy/eclipse-pde-partial-idea/releases/tag/v1.3.6.2)
This is pre-release, will not public via JetBrains Marketplace, you must download it from release of Github and manually install it.
